src/TextInput/index.js

```diff
@@ -185,8 +205,16 @@ class TextInput extends React.Component<Props, State> {
                 styles[size],
               ]}
             />
-
-            {suffix != null && <Suffix>{suffix}</Suffix>}
+            {Platform.OS !== 'web' && (
+              <ClearButton
+                onPress={this.clearValue}
+                focused={focused}
+                value={value}
+              />
+            )}
+            {suffix != null && Platform.OS === 'web' && (
```

Comment:
So that means I cannot see the service logo anymore. I think, even though it's not in the design, we had a use case for a payment form. But now that I think of it, it was actually prefixing the input field, and not suffixing. So maybe we can leave it like this.
@tbergq thoughts?

Reply:
>  it was actually prefixing the input field, and not suffixing.

I had the same feelings, that's why removed it from mobile.